scripts/jenkins-support.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+function connect_rally_server_run_test
+{
+    # Connect to rally server and run tests
+    ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+    cloud=$1
+    image_name=$2
+
+    source .openrc
+    # Check if jeos image already exist on cloud
+    openstack image show -c name --format value $image_name
+    image_exist=$?
+    # Upload jeos image
+    if [[ $image_exist != 0 ]]; then
+       curl -s http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/JeOS/images/SLES12-SP2-JeOS-for-OpenStack-Cloud.x86_64-1.2.0-Build13.308.qcow2| \
+       openstack image create --public --disk-format qcow2 $image_name
```

Comment:
1. This link does not exist. You hardcoded the build number. And as soon as the new build comes out the old one is deleted.

2. Even if you hardcode it there is currently a 302 redirect. So u need to add -L parameter to curl. But that is probably only from my machine? need to doublecheck.